sqlx-core/src/sqlite/options/mod.rs

```diff
@@ -367,4 +369,13 @@ impl SqliteConnectOptions {
         self.row_channel_size = size;
         self
     }
+
+    /// Sets the [`vfs`](https://www.sqlite.org/vfs.html) parameter of the database connection.
+    ///
+    /// The default value is empty, and sqlite will use the default VFS object dependeing on the
+    /// operating system.
+    pub fn vfs(mut self, vfs_name: impl AsRef<str>) -> Self {
+        self.vfs = Some(Cow::Owned(vfs_name.as_ref().to_string()));
```

Comment:
`impl Into<Cow<'static, str>>` would avoid copies when the user passes `&'static str` or `String`